code/modules/hippie/code/modules/admin/verbs/smite.dm

```diff
@@ -0,0 +1,65 @@
+//lightning effect definition moved to hippie module path is hippie/object/effects/holy.dm
+
+/client/proc/cmd_smite(mob/living/M in mob_list)
+	set category = "Fun"
+	set name = "Smite"
+	if(!holder || !check_rights(R_FUN))
+		return
+
+	if(!isliving(M))
+		usr << "This can only be used on instances of type /mob/living"
+		return
+
+	var/options = list("Brute","Burn","Toxin","Oxygen","Clone","Brain","Stamina","Heal","Gib","Cancel")
+	var/requiresdam = options - list("Heal","Gib","Cancel")
+
+	var/confirm = null
+	var/damtype_word = null
+	var/dam = 0
+
+	confirm = input(src, "Really smite [M.name]([M.ckey])?", "Divine Retribution") in list("Yeah", "Nah")
+	if(confirm == "Nah")
+		return
+
+	damtype_word = input(src, "What kind of damage?", "PUT YOUR FAITH IN THE LIGHT") in options
+
+	if(damtype_word in requiresdam)
+		dam = input(src, "How much damage?", "THE LIGHT SHALL BURN YOU") as num
+		if(!dam)
+			return
+
+	if(damtype_word == "Cancel")
+		return
+
+	var/obj/effect/holy/lightning/L = new /obj/effect/holy/lightning()
+	L.start(M)
+
+	spawn(10)
```

Comment:
So you want me to make a proc for this? because the code in the spawn is not a proc and the spawn is being used as a 1 second delay so the lighting effect can hit the person then it does something which looks better.